Applications/APRS/aprs.py

```diff
@@ -221,29 +221,24 @@ def sendPositions(stations, socket):
 
             if node != destNode:
                 # APRS string is for remote RF node
-                aprsPosition = dataTypeIdent +\
-                               latString +\
-                               latitudeDirection +\
-                               symbolTable +\
-                               lonString +\
-                               longitudeDir +\
-                               symbol
-
-                positionString = node +\
-                                 '>' +\
-                                 destAddress +\
-                                 ',' +\
-                                 qConstruct +\
-                                 ',' +\
-                                 destNode +\
-                                 ':' +\
-                                 aprsPosition +\
-                                 '.../' +\
-                                 speed +\
-                                 '/A=' +\
-                                 altitude +\
-                                 comment +\
-                                 '\r'
+                aprsPosition = ''.join([
+                    dataTypeIdent,
+                    latString,
+                    latitudeDirection,
+                    symbolTable,
+                    lonString,
+                    longitudeDir,
+                    symbol])
+
+                positionString = '{}>{},{},{}:{}.../{}/A={}{}\r'.format(
```

Comment:
Interesting use of a format(), very close to how `aprslib` does it. This code is going to be replaced by `aprslib` functions in the future FYI. I do want @kb1lqd to see this. Nifty formatting. WAYY cleaner than how I did it prior.